plugins/polio/models.py

```diff
@@ -874,3 +874,26 @@ class Meta:
 
     def __str__(self):
         return str(self.event)
+
+
+class VaccineAuthorization(SoftDeletableModel):
+    STATUS = (
+        ("ongoing", "Ongoing"),
+        ("signature", "Sent for signature"),
+        ("validated", "Validated"),
+        ("expired", "Expired"),
+    )
```

Comment:
What's the benefice of this ? it's just creating a class for the choices outside of the model class ?

Reply:
I think the benefit is more readability, and a common way of doing stuff in the project. Otherwise I think both ways are officially supported by Django, see https://docs.djangoproject.com/en/3.1/ref/models/fields/#enumeration-types

Reply:
[Enumeration](https://docs.python.org/3/library/enum.html#module-enum) types are cool 😎

`models.TextChoices` gives you a small API useful in tests and in the code:

```python
Model.Status.choices
# [('A', 'Foo'), ('B', 'Bar'), ('C', 'Baz')]

dict(Model.Status.choices)

Model.Status.values
# ['A', 'B', 'C']

Model.Status.labels
# ['Foo', 'Bar', 'Baz']

Model.Status.A.value
# 'A'

Model.Status.A.label
# 'Foo'

str(Model.Status.A) == Model.Status.A.value
```